rust/feature-flags/src/api/endpoint.rs

```diff
@@ -60,6 +92,10 @@ pub async fn flags(
 ) -> Result<Json<ServiceResponse>, FlagError> {
     let request_id = Uuid::new_v4();
 
+    if method != Method::POST {
```

Comment:
Hmm, ideally this would check that it's a `GET` response and return unsupported for `DELETE`, `PATCH`, etc. But I'm ok with this for now if this is what DECIDE does.